docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+services:
+  ganache:
+    container_name: ganache
+    image: trufflesuite/ganache-cli
+
+    command: --host 0.0.0.0 --port 7545
```

Comment:
```suggestion
    command: --host 0.0.0.0 --port 7545 --mnemonic "lighten crowd lavish spectacular bustling saw tasteless nauseating panoramic mixed absorbing past"
```

Reply:
It would be nice if you could add --gasLimit flag with some high value like 20000000 to have safety when people would try to create bigger objects.